src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  return (
+    <div className="CommentList">
+      {comments.map(
+        comment =>
+          comment && <CommentInfo key={comment.id} comment={comment} />,
+      )}
```

Comment:
```suggestion
      {comments.map(
        comment => <CommentInfo key={comment.id} comment={comment} />,
      )}
```
In what case you can "missing comment" in comments array?